### PR TITLE
Sync Sensor Speeds & Reset UWP to 0

### PR DESCRIPTION
**Pull request by jamesmontemagno**
Restore the default report interval to release resources while the sensor is not in use

### CODE REVIEW

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Xamarin.Essentials/Accelerometer/Accelerometer.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Accelerometer/Accelerometer.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Compass/Compass.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Compass/Compass.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Gyroscope/Gyroscope.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Gyroscope/Gyroscope.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Magnetometer/Magnetometer.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Magnetometer/Magnetometer.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/OrientationSensor/OrientationSensor.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/OrientationSensor/OrientationSensor.uwp.cs) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C27%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201807272335420157-419%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hi%2Fit5gUyuUNz41gISfwM4vd0%2FXmOIudTCmt8Jk8XhI%3D&st=2018-07-27T23%3A31%3A43Z&se=2018-08-27T23%3A36%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Xamarin.Essentials/Accelerometer/Accelerometer.android.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Accelerometer/Accelerometer.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Accelerometer/Accelerometer.ios.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Accelerometer/Accelerometer.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Accelerometer/Accelerometer.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Accelerometer/Accelerometer.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Compass/Compass.android.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Compass/Compass.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Compass/Compass.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Compass/Compass.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Gyroscope/Gyroscope.android.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Gyroscope/Gyroscope.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Gyroscope/Gyroscope.ios.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Gyroscope/Gyroscope.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Gyroscope/Gyroscope.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Gyroscope/Gyroscope.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Magnetometer/Magnetometer.android.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Magnetometer/Magnetometer.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Magnetometer/Magnetometer.ios.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Magnetometer/Magnetometer.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Magnetometer/Magnetometer.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Magnetometer/Magnetometer.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/OrientationSensor/OrientationSensor.android.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/OrientationSensor/OrientationSensor.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/OrientationSensor/OrientationSensor.ios.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/OrientationSensor/OrientationSensor.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/OrientationSensor/OrientationSensor.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/OrientationSensor/OrientationSensor.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/SensorSpeed.android.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Types/SensorSpeed.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/SensorSpeed.ios.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Types/SensorSpeed.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/SensorSpeed.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/uwp-sensor-interval-reset/Xamarin.Essentials/Types/SensorSpeed.uwp.cs) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C28%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201807280002395422-419%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=c8zLZ6Lsbf6Px9bGbcwiUdzSrxOiPV2iVf18d6XQbk4%3D&st=2018-07-27T23%3A58%3A39Z&se=2018-08-28T00%3A03%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Xamarin.Essentials/Types/SensorSpeed.uwp.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿namespace Xamarin.Essentials
+{
+    static class SensorSpeedExtensions
+    {
+        internal static uint ToPlatform(this SensorSpeed sensorSpeed)
```
Comment (jerone) [inline Xamarin.Essentials/Types/SensorSpeed.uwp.cs]: Shouldn't this not return `8`, `22` & `33`.

Comment (jamesmontemagno) [reply Xamarin.Essentials/Types/SensorSpeed.uwp.cs]: For the compass, yes it is, but not for other sensors as the documentation don't specify any of it, so we should keep all three platforms the same.

Comment (jerone) [reply Xamarin.Essentials/Types/SensorSpeed.uwp.cs]: Ok sorry, then I probably misunderstood OrientationSensor, magnetometer, gyroscope & accelerometer.

Comment (jamesmontemagno) [reply Xamarin.Essentials/Types/SensorSpeed.uwp.cs]: Yeah, so for instance accelerometer: https://docs.microsoft.com/en-us/uwp/api/windows.devices.sensors.accelerometer.reportinterval

Where as compass has specifics to use: https://docs.microsoft.com/en-us/uwp/api/windows.devices.sensors.compass.reportinterval

So just restructuring to make them all the same across devices if no info is available.

**Pull request merged**